app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -53,6 +53,13 @@ def find_by_installable_errata(_key, operator, value)
             { :conditions => "#{::Host::Managed.table_name}.id IN (#{facets.pluck(:host_id).join(',')})" }
           end
         end
+
+        def in_content_view_environment(content_view = nil, lifecycle_environment = nil)
```

Comment:
When 1 or both can be nil, I think the number of arguments diminishes the reason to do it given you have to pass nil which isn't obvious. What I might ask for instead of changing the method signature is to use keywords where the nil is since this is Ruby 2, e.g. 

```
Host.in_content_view_environment(:lifecycle_environment => env)
```

The above should work out of the both without changing the method signature.